clicks.view.lkml

```diff
@@ -394,13 +394,15 @@ view: clicks {
   # This section contains fields that are duplicated across over view files in this project
 
   dimension: is_government {
+    description: "Yes if the IP address maps to a known BC Government network."
     # the filter is put in this view because the IP is defined here in this view
     type: yesno
     # the filter is checking to see if the IP is in the gov network
     sql: ${ip_address} LIKE '142.22.%' OR ${ip_address} LIKE '142.23.%' OR ${ip_address} LIKE '142.23.255%' OR ${ip_address} LIKE '142.24.%' OR ${ip_address} LIKE '142.31.%' OR ${ip_address} LIKE '142.31.255%' OR ${ip_address} LIKE '142.25.%' OR ${ip_address} LIKE '142.26.%' OR ${ip_address} LIKE '142.27.%' OR ${ip_address} LIKE '142.28.%' OR ${ip_address} LIKE '142.29.%' OR ${ip_address} LIKE '142.30.%' OR ${ip_address} LIKE '142.32.%' OR ${ip_address} LIKE '142.33.%' OR ${ip_address} LIKE '142.34.%' OR ${ip_address} LIKE '142.35.%' OR ${ip_address} LIKE '142.36.%' ;;
   }
 
   dimension: ip_address {
+    description: "The user's IP address, with the final octet replaced with \"1\" to remove personally identifiable information."
```

Comment:
I think that ends up being an .x when snowplow's model is added. So when working with it in looker it will be 123.345.123.x.

Reply:
it is 1 my mistake